### ISSUE TITLE
Edit event dates displays error in dashboard but no info is provided

### ISSUE THREAD

**Issue by abelsromero** [alfio-event/alf.io#1039]
**To Reproduce**
After updating a pre-existig event with new dates, the following message appears in the dashboard but no indication of what the error is about is shown.
![image](https://user-images.githubusercontent.com/5781153/145267035-12393c49-ee63-4588-9327-e367bd8ee4b5.png)

**Expected behavior**
If some data validation shows issues, I'd expect to be displayed in the dashboard of the general event view.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: Firefox
 - Version: 2.0-M2-SNAPSHOT

**Additional context**
Basically, we want to update a past cancelled event (for COVID) with metadata for new event, keeping all sold tickets and other information for the current new. So far, other than this issue, the only other problem is we cannot update the url, but I think we can live with it.

**Comment by cbellone** [comment]
Hi,
thanks for letting us know.

Please double check for expired fixed-size (hidden or public) ticket categories. That's usually the case. We'll improve that warning message in the next versions (M4+).

2.0-M2 is an old and unmaintained version, so it doesn't make sense for us to fix the bug there. Instead, it would be great if you could try to run [2.0-M4.RC2](https://github.com/alfio-event/alf.io/releases/tag/2.0-M4.RC2) on a **copy of the database**, just to make sure that this issue has been fixed.

**Comment by abelsromero** [comment]
We migrated and after some ticket dates update we see this again. Is there some way to find a hint?
